railties/test/application/rake_test.rb

```diff
@@ -99,7 +99,7 @@ def test_should_not_eager_load_model_for_rake
     end
 
     def test_code_statistics_sanity
-      assert_match "Code LOC: 5     Test LOC: 0     Code to Test Ratio: 1:0.0",
+      assert_match "Code LOC: 7     Test LOC: 0     Code to Test Ratio: 1:0.0",
```

Comment:
@cristianbica Somehow I had to change this line otherwise the tests did not pass. Do you have any idea why that is? It is because I added the `application_job.rb` file?